pyvistaqt/plotting.py

```diff
@@ -161,28 +170,28 @@ class QtInteractor(QVTKRenderWindowInteractor, BasePlotter):
     render_signal = pyqtSignal()
     key_press_event_signal = pyqtSignal(vtk.vtkGenericRenderWindowInteractor, str)
 
+    # pylint: disable=too-many-arguments
     def __init__(
         self,
-        parent=None,
-        title=None,
-        off_screen=None,
-        multi_samples=None,
-        line_smoothing=False,
-        point_smoothing=False,
-        polygon_smoothing=False,
-        auto_update=5.0,
-        **kwargs
-    ):  # pylint: disable=too-many-arguments
+        parent: MainWindow = None,
+        title: str = None,
+        off_screen: bool = None,
+        multi_samples: int = None,
+        line_smoothing: bool = False,
+        point_smoothing: bool = False,
+        polygon_smoothing: bool = False,
+        auto_update: Union[float, bool] = 5.0,
+        **kwargs: Any
+    ) -> None:
```

Comment:
It's great that `sphinx` can build the docs from this and we don't have to include the type definitions within the doc string!

Reply:
@akaszynski That is the cool work of @tk0miya .